src/open_clip/model.py

```diff
@@ -209,6 +228,12 @@ def encode_text(self, text, normalize: bool = False):
     def forward(self, image, text):
         image_features = self.encode_image(image, normalize=True)
         text_features = self.encode_text(text, normalize=True)
+        if self.output_dict is not None:
+            return {
```

Comment:
why is this None and not bool? bool attr should work fine with torchscript

Reply:
@gpucce tried changing that here: https://github.com/mlfoundations/open_clip/pull/373